lib/ephemeral/app/resources/file.rb

```diff
@@ -5,7 +5,10 @@ class File < Grape::API
       
       desc 'Creates a new file'
       params do
-        requires :file, desc: 'File to upload'
+        group :file, type: Hash do
+          requires :filename, type: String, desc: 'File name', values: ["README.md"]
+          requires :type, type: String, desc: 'File type', values: ["text/plain"]
```

Comment:
For the :filename and :type you should not be testing for those values as they can be anything the user uploads; so just cut the `, values: [...]` bit